conda/meta.yaml.in

```diff
@@ -0,0 +1,78 @@
+package:
+  name: adcc
+  version: @ADCC_VERSION@
+
+source:
+    git_url: git@github.com:adc-connect/adcc.git
```

Comment:
I guess this needs to be replaced by the source directory because `adcc` is now a submodule of `adcc-conda` and the correct version is pinned and checked out there?

Reply:
Yes, I think you are right.

Reply:
I changed it, seems to work.